public/javascripts/gtresolution.js

```diff
@@ -339,67 +374,67 @@ $(document).ready(function () {
         var markerIndex = pano.labelMarkers.length - 1;
         //CLICK
         google.maps.event.addListener(pano.labelMarkers[markerIndex], 'click', function () {
-            //bring label to the front
+            // Bring label to the front
             maxZIndex++;
             pano.labelMarkers[markerIndex].setOptions({zIndex: maxZIndex});
-            //hide all other popovers
+            // Hide all other popovers
             for (var i = 0; i < pano.labelMarkers.length; i++) {
-                if (i != markerIndex) {
+                if (i !== markerIndex) {
                     $('#' + pano.labelMarkers[i].getId()).popover('hide');
                     var closeMarker = mapMarkers.find(mkr => mkr.meta.label_id === pano.labels[i].label_id);
                     closeMarker.setOptions({fillColor: colorMapping[closeMarker.meta.label_type].fillStyle});
                     closeMarker.clicked= false;
                 }
             }
-            //empasize/deemphasize corresponding label on map
+            // Empasize/deemphasize corresponding label on map
             var marker = mapMarkers.find(mkr => mkr.meta.label_id === label.label_id);
             if (marker.clicked) {
                 marker.setOptions({fillColor: colorMapping[marker.meta.label_type].fillStyle}); marker.clicked = false
             } else {marker.setOptions({fillColor: '#ff38fb'}); marker.clicked = true;}
-            //create and open popover for label
+            // Create and open popover for label
             createPopover(pano, label, status);
         });
 
-        //CHANGE IN POV
+        // CHANGE IN POV
         google.maps.event.addListener(pano.gsv_panorama, 'pov_changed', function () {
-            // Popover follows marker when POV is changed
+            // Popover follows marker when POV is changed (no need to update if we are just doing severity update)
             createPopover(pano, label, status);
             $("#" + id).popover('hide');
-            var closeMarker = mapMarkers.find(mkr => mkr.meta.label_id === label.label_id);
-            closeMarker.setOptions({fillColor: colorMapping[closeMarker.meta.label_type].fillStyle});
-            closeMarker.clicked= false;
+            if (status !== "One Turker") {
```

Comment:
It is actually explained in the comment that I updated, just 3 lines before that line. It just isn't immediately visible when looking at the diff since there is a bunch of stuff in between.